src/test/java/spoon/test/main/MainTest.java

```diff
@@ -177,6 +178,9 @@ public void scan(CtElement element) {
 					if (reference.getParameters().get(i) instanceof CtArrayTypeReference && ((CtArrayTypeReference) reference.getParameters().get(i)).getComponentType() instanceof CtTypeParameterReference) {
 						continue;
 					}
+					if (executableDeclaration instanceof CtLambda) {
```

Comment:
could you add a comment line to explain why we can not make the assertion for Lambdas?

Reply:
I do not know whether we cannot make assertions about lambdas. Lambdas are too different (comparing to method) and they are not supported yet by this test. So may be there is necessary to write lambda specific assertion. But I did not understood the purpose of this test, so I would rather let You to fix it.